ajax/libs/alertify.js/package.json

```diff
@@ -3,7 +3,7 @@
   "filename": "alertify.min.js",
   "version": "0.5.0",
   "description": "Alertify is an unobtrusive customizable JavaScript notification system.",
-  "homepage": "http://fabien-d.github.io/alertify.js/",
+  "homepage": "https://fabien-d.github.io/alertify.js/",
```

Comment:
Please make this change in another commit, thanks.

Reply:
What do you mean in another commit? I made it in `Update alertify-js package.json to use git autoupdate  …`

Reply:
Use new commit to do it.